### PR TITLE
Adding a screenshot to check the passing of the test

### LINKED ISSUE #9: Take a screenshot after passing a test
Write a decorator that takes a screenshot after passing a test

### PR DESCRIPTION
**Pull request by JuliaMetel**

### CODE REVIEW

File: test_gender_selection.py [hunk 1]
```diff
@@ -1,33 +1,58 @@
...
 class TestClass:
+
     def test_gender_selection_female(self, driver):
         driver.get("https://webglsamples.org/collectibles/index.html")
-        elem = driver.find_element('id', 'dollbaseFemale')
-        elem.click()
-        WebDriverWait(driver, 5).until(EC.presence_of_element_located(('id', 'canvas')))
-
+        driver.find_element('id', 'dollbaseFemale').click()
+        test_path = 'test_gender_selection_female_test.png'
```
Comment (IamVladislav) [inline test_gender_selection.py]: Please, move lines 34-43 into function with name like `wait_page_stable` or `wait_animation_end` and reuse it in both tests

File: test_gender_selection.py [hunk 2]
```diff
@@ -1,33 +1,58 @@
...
+        test_path = 'test_gender_selection_male_test.png'
+        driver.save_screenshot(test_path)
+        for x in range(10):
+            time.sleep(0.1)
+            path = 'test_gender_selection_male.png'
+            driver.save_screenshot(path)
+            if is_images_equal(test_path, path):
+                break
+            driver.save_screenshot(test_path)
+        os.remove(test_path)
```
Comment (IamVladislav) [inline test_gender_selection.py]: And what about second image? Why don't you delete it?

Comment (JuliaMetel) [reply test_gender_selection.py]: The TK test should end with a screenshot. In the implementation, it makes no sense to take this screenshot and make it again. Will be corrected in task 12.

File: test_gender_selection.py (hunk 1 above)
Comment (JuliaMetel) [reply test_gender_selection.py]: Fixed

**Pull request merged**